roles/edpm_update/tasks/containers.yml

```diff
@@ -100,6 +100,17 @@
     - edpm_update
   when: '"nova" in edpm_update_running_services'
 
+- name: Updates configs for edpm_nova role
+  ansible.builtin.include_role:
+    name: osp.edpm.edpm_nova
+    tasks_from: configure.yml
+    apply:
+      become: true
```

Comment:
Is this actually necessary? I can see there are already some tasks in the `configure.yml` that have `become: true` as required. ie:
https://github.com/openstack-k8s-operators/edpm-ansible/blob/main/roles/edpm_nova/tasks/configure.yml#L9

Reply:
I put it here as all other tasks are using it :)
I am not sure if this won't break resulted permissions.
How does edpm service executed by ansible runner, does it enfore become?
We should call this the same way we do for greenfield deployments

Reply:
I don't think it's needed. It's not set when configure.yml is included from edpm_nova/tasks/main.yml, it's not set in playbooks/nova.yml, and there is nothing in how openstack-operator creates the runner jobs that sets a universal become:true.